test/flutter_inapp_purchase_test.dart

```diff
@@ -1400,7 +1400,7 @@ void main() {
         });
 
         FlutterInappPurchase(FlutterInappPurchase.private(
-            FakePlatform(operatingSystem: "ios"),
+            FakePlatform(operatingSystem: "android"),
```

Comment:
Here is the unit test for Android, so I just changed `ios` to `android`.

https://github.com/dooboolab/flutter_inapp_purchase/blob/24b290cff01d99494e7a38fb27373027acf8cd07/test/flutter_inapp_purchase_test.dart#L1396-L1405